hms-app/src/actions/UserActions.tsx

```diff
@@ -0,0 +1,75 @@
+const core_url = process.env.REACT_APP_CORE_URL
+
+export const getListOfUsers = () => {
+  return fetch(`${core_url}/users`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((data) => {
+      return data.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const getUserDetails = (userID: string) => {
+  return fetch(`${core_url}/user/${userID}`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((data) => {
+      return data.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const deleteUser = (userID: string) => {
+  return fetch(`${core_url}/hackathon/${userID}`, {
+    method: 'DELETE',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((response) => {
+      return response.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const editUser = (
+  contextID: string,
+  user: {
+    lastName: string
+    firstName: string
+    emailAddress: string
+    roles: any[]
+    imageUrl: string
+  },
+  skills: string[]
```

Comment:
so a similar thing here, contextID is a bit misleading and you won't know just by looking at it what value should go there.  So userID would be better.  In addition this could all just be one user object, or all individual fields instead of the mix